src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5123,4 +5123,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_TypeReserved" xml:space="preserve">
     <value>The type name '{0}' is reserved to be used by the compiler.</value>
   </data>
+  <data name="ERR_FieldsInRoStruct" xml:space="preserve">
+    <value>Auto-implemented instance properties in readonly structs must be readonly.</value>
+  </data>
+  <data name="ERR_AutoPropsInRoStruct" xml:space="preserve">
+    <value>Field-like events are not allowed in readonly structs.</value>
```

Comment:
tests only test that the right error is produced. The actual error is the right one. It is the resource that is mismatched.
I've fixed the resource. Now it should be correct :)

---
In reply to: [121741542](https://github.com/dotnet/roslyn/pull/20142#discussion_r121741542) [](ancestors = 121741542)